docs/reference/auditing.md

```diff
@@ -16,38 +16,240 @@ lakeFS Cloud
+1. [Reach out to customer success](mailto:support@treeverse.io?subject=ARN to use for audit logs) and provide this ARN. Once receiving the ARN role, an access point will be created and you should get in response the following details:
+   1. S3 Bucket (e.g. `arn:aws:s3:::lakefs-audit-logs-us-east-1-production`)
+   2. S3 URI to an access point (e.g. `s3://arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>`)
+   3. Access Point alias. You can use this alias instead of the bucket name or Access Point ARN to access data through the Access Point. (e.g. `lakefs-logs-<generated>-s3alias`)
+3. Update your IAM Role policy and trust policy if required
+
+A minimal example for IAM policy with 2 lakeFS installations in 2 regions (`us-east-1`, `us-west-2`):
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::lakefs-audit-logs-us-east-1-production",
+                "arn:aws:s3:::lakefs-audit-logs-us-east-1-production/*",
+                "arn:aws:s3:::lakefs-logs-<generated>-s3alias/*",
+                "arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>",
+                "arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>/*"
+            ],
+            "Condition": {
+                "StringLike": {
+                    "s3:prefix": [
+                        "etl/v1/data/region=<region_a>/organization=org-<organization>/*",
+                        "etl/v1/data/region=<region_b>/organization=org-<organization>/*"
+                    ]
+                }
+            }
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:GetObjectVersion"
+            ],
+            "Resource": [
+                "arn:aws:s3:::lakefs-audit-logs-us-east-1-production",
+                "arn:aws:s3:::lakefs-audit-logs-us-east-1-production/etl/v1/data/region=<region_a>/organization=org-<organization>/*",
+                "arn:aws:s3:::lakefs-audit-logs-us-east-1-production/etl/v1/data/region=<region_b>/organization=org-<organization>/*",
+                "arn:aws:s3:::lakefs-logs-<generated>-s3alias/*",
+                "arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>/object/etl/v1/data/region=<region_a>/organization=org-<organization>/*",
+                "arn:aws:s3:us-east-1:924819537486:accesspoint/lakefs-logs-<organization>/object/etl/v1/data/region=<region_b>/organization=org-<organization>/*"
+            ]
+        },
+        {
+            "Action": [
+                "kms:Decrypt"
+            ],
+            "Resource": [
+                "arn:aws:kms:us-east-1:924819537486:key/a2ad86b0-99a3-4f56-8141-bc07ce7b0f08"
```

Comment:
I suggest to add a section under the information you get from customer success and in here replace the account id and key id with a placeholder.